tests/garage/experiment/test_experiment_wrapper.py

```diff
@@ -0,0 +1,27 @@
+import base64
+import unittest.mock
+
+import cloudpickle
+
+from garage.experiment import SnapshotConfig
+from garage.experiment.experiment_wrapper import run_experiment
+
+
+class TestExperimentWrapper(unittest.TestCase):
+    @staticmethod
+    def method_call(snapshot_config, variant_data):
+        assert isinstance(snapshot_config, SnapshotConfig)
```

Comment:
it it doesn't need `self`, you can just make this a method in the module (outside the class)